README.md

```diff
@@ -1 +1,49 @@
-# vmpooler-dns-google-clouddns
\ No newline at end of file
+# vmpooler-dns-gcp
+
+- [vmpooler-dns-gcp](#vmpooler-dns-gcp)
+  - [Requirements](#requirements)
+  - [Usage](#usage)
+  - [Update the Gemfile Lock](#update-the-gemfile-lock)
+  - [Releasing](#releasing)
+  - [License](#license)
+
+## Requirements
+
+1. A Google Cloud Project with the [Cloud DNS](https://cloud.google.com/dns/) enabled.
```

Comment:
```suggestion
1. A Google Cloud Project with the [CloudDNS](https://cloud.google.com/dns/) enabled.
```
I am fairly sure there is no space...

Reply:
To my surprise the link to the docs on that line actually does have a space between them